docs/running.md

```diff
@@ -18,9 +18,11 @@ sudo docker run \
 
 cAdvisor is now running (in the background) on `http://localhost:8080/`. The setup includes directories with Docker state cAdvisor needs to observe.
 
-**Note**: If docker daemon is running with [user namespace enabled](https://docs.docker.com/engine/reference/commandline/dockerd/#starting-the-daemon-with-user-namespaces-enabled),
-You need to add `--userns=host` option in order for cAdvisor to monitor Docker containers,
+**Note**: 
+- If docker daemon is running with [user namespace enabled](https://docs.docker.com/engine/reference/commandline/dockerd/#starting-the-daemon-with-user-namespaces-enabled),
+you need to add `--userns=host` option in order for cAdvisor to monitor Docker containers,
 otherwise cAdvisor can not connect to docker daemon.
+- If cadvisor scrapes `process metrics` by set flag `--disable_metrics`, you need to add `--pid=host --privileged` to get `/proc/pid/fd` path in host.
```

Comment:
what are --pid=host --privileged flags for?  Are these `docker run` flags?  If possible, we should make it clear what setups this applies to.

Reply:
okay, changed